make_release.py

```diff
@@ -78,18 +86,21 @@ def announce_new_release(release_name):
         print("update conda-forge feedstock steps:", ', '.join(f.__name__ for f, _ in update_feedstock_steps))
         print("or")
         print("Usage: {} -a|--announce release_name".format(argv[0]))
-        print("Usage: {} -m release_name".format(argv[0]))
+        print("Usage: {} -m|--meta release_name".format(argv[0]))
+        print("Usage: {} -l|--logs release_name".format(argv[0]))
```

Comment:
I am not a fan of this proliferation of commands. Can't this merge changelog be one normal "step" of the release process? I mean, is it realistic to do this step within the usual sequence of release steps, or will does it **need** to be a totally separate command? I did **not** think it through. Just asking the question.